### PR TITLE
feat(crux): add secrets to template

### PR DESCRIPTION
**Pull request by peterambrus3**
From now you can add your secrets to the template.

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]
Looks good to me 👏🏻

File: web/crux/src/services/template.file.service.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ type TemplateConfig = {
   volumes: TemplateVolume[]
   ports: Port[]
   environment: UniqueKeyValue[]
+  secrets: UniqueKey[]
```
Comment (balanceee) [inline web/crux/src/services/template.file.service.ts:35]: I think it should be UniqueSecretKey.

File: web/crux/src/services/template.file.service.ts [hunk 2]
```diff
@@ -44,6 +45,7 @@ export interface TemplateImage {
   capabilities: UniqueKeyValue[]
   config: TemplateContainerConfig
   environment: UniqueKeyValue[]
+  secrets: UniqueKey[]
```
Comment (balanceee) [inline web/crux/src/services/template.file.service.ts:48]: Same as above

File: web/crux/src/services/template.file.service.ts (hunk 1 above)
Comment (peterambrus3) [reply web/crux/src/services/template.file.service.ts]: true 🤦‍♂️

**Review by balanceee — APPROVED** [review]
LGTM

**Pull request merged**